app/assets/stylesheets/homepage.css.scss

```diff
@@ -6,7 +6,15 @@
 #nav-bar {
     position:absolute;
     background-color: rgba(0,0,0,.4);
-    padding: 10px;
+    padding-top: 10px;
+
+  -moz-box-shadow:    inset 0px 5px 20px 0px rgba(0,0,0,0.3);
+  -webkit-box-shadow: inset 0px 5px 20px 0px rgba(0,0,0,0.3);
```

Comment:
Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`